<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManufDialog</class>
 <widget class="QDialog" name="ManufDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MAC Address Blocks</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="ouiRadioButton">
       <property name="toolTip">
        <string>Search MAC address or address prefix. Special purpose bits are masked.</string>
       </property>
       <property name="text">
        <string>MAC Address</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">radioButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="vendorRadioButton">
       <property name="toolTip">
        <string>Search vendor name using a case-insentitive regular expression.</string>
       </property>
       <property name="text">
        <string>Vendor Name</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">radioButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkShortNameButton">
       <property name="toolTip">
        <string>Show short name column.</string>
       </property>
       <property name="text">
        <string>Short name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="hintLabel">
     <property name="text">
      <string notr="true">&lt;small&gt;&lt;i&gt;A hint.&lt;/i&gt;&lt;/small&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="manufLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="manufTableView">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>140</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManufDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManufDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="radioButtonGroup"/>
 </buttongroups>
</ui>
